apis/management.cattle.io/v3/cluster_types.go

```diff
@@ -95,6 +95,7 @@ type ClusterSpecBase struct {
 	EnableNetworkPolicy                  *bool                          `json:"enableNetworkPolicy" norman:"default=false"`
 	EnableClusterAlerting                bool                           `json:"enableClusterAlerting" norman:"default=false"`
 	EnableClusterMonitoring              bool                           `json:"enableClusterMonitoring" norman:"default=false"`
+	WindowsPreferedCluster               bool                           `json:"windowsPreferedCluster" norman:"default=false,noupdate"`
```

Comment:
no need to set default=false; all booleans in go get defaulted to false w/o it

Reply:
Updated